Riot/Modules/Analytics/PostHogAnalyticsClient.swift

```diff
@@ -20,30 +20,31 @@ import AnalyticsEvents
 /// An analytics client that reports events to a PostHog server.
 class PostHogAnalyticsClient: AnalyticsClientProtocol {
     /// The PHGPostHog object used to report events.
-    private var postHog: PHGPostHog?
+    private var postHog: PostHogSDK?
     
     /// Any user properties to be included with the next captured event.
     private(set) var pendingUserProperties: AnalyticsEvent.UserProperties?
     
     static let shared = PostHogAnalyticsClient()
     
-    var isRunning: Bool { postHog?.enabled ?? false }
+    var isRunning: Bool { postHog != nil }
```

Comment:
you can use the `isOptOut` method

Reply:
Not sure if lot better? would be a not isOptOut() and as it's optional could be a bit harder to read.

Reply:
`postHog` can be not nil and is still internally disabled, so `isOptOut` is the method to call.
`!isOptOut()` == enabled
`isOptOut()` == disabled.

Reply:
Updated, let me know if ok for you